db/system_keyspace.cc

```diff
@@ -2322,6 +2322,8 @@ future<> system_keyspace::make(
         co_await db.create_local_system_table(table, maybe_write_in_user_memory(table), erm_factory);
         co_await db.find_column_family(table).init_storage();
     }
+
+    db.find_keyspace(NAME).add_user_type(sstableinfo_type);
```

Comment:
Does this get persisted to the disk, into the schema tables?

Reply:
The call is required to have the UDT defined in `system_schema.types` table. Otherwise it is not there.